README.md

```diff
@@ -254,6 +254,22 @@ or
 pytest test/integration/trino.dbtspec
 ```
 
+### Release process
+
+Before doing a release dbt's version requires updating.
+In order to bump dbt-trino and dbt-core version run:
+
+```sh
+bumpversion --config-file .bumpversion-dbt.cfg patch --new-version <new-version>
+bumpversion --config-file .bumpversion.cfg patch --new-version <new-version> --allow-dirty
+```
+
+Additionally bump dbt-core version in dbt testing image `docker/dbt/Dockerfile`.
+
+Next step is to merge bump commit and making sure that test suite pass.
+
+Finally to release `dbt-trino` to PyPi and GitHub trigger release workflow `release.yml`.
```

Comment:
How does a maintainer trigger the "release.yml" workflow?

Reply:
Via GHA: Actions --> release --> Run workflow. It's triggered using `workflow_dispatch` event type. https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#workflow_dispatch